docs/Frequently Asked Questions/Multiple Process Items/index.md

```diff
@@ -0,0 +1,98 @@
+# Multiple Process Items
+
+The SmartDeviceLink Android library uses multiple process and there are some items that should be understood about why that is necessary and precautions to take while handling that situation. 
+
+## Why does the router service run in its own process?
+
+The router service is designed to live outside the normal lifecycle of the app integrating the SDL framework. The different process allows a level of security to cut off access to the hosting application's data. It also allows the router service to not interfere with the hosting applications runtime; this means if the router service unexpectedly stops or crashes, it will not take down the entire app. This relationship also works in the opposite direction, which is important to maintain a good user experience when apps are connected through a router service.
```

Comment:
```suggestion
The router service is designed to live outside the normal lifecycle of the app integrating the SDL framework. The different process allows a level of security to cut off access to the hosting application's data. It also allows the router service to not interfere with the hosting application's runtime; this means if the router service unexpectedly stops or crashes, it will not take down the entire app. This relationship also works in the opposite direction, which is important to maintain a good user experience when apps are connected through a router service.
```